collections/_article/update-on-google-forge-2024.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Update on the Collaboration with Google and The Forge"
+excerpt: "The collaboration with Google and The Forge has concluded successfully!"
+categories: ["news"]
+author: Clay John
+image: /storage/blog/covers/android-and-vulkan.webp
+date: 2023-12-19 15:00:00
+---
+
+Our [collaboration with Google and The Forge](https://godotengine.org/article/collaboration-with-google-forge-2023) has achieved its original goal of improving performance in the Vulkan backend and enhancing our Vulkan API usage. The work started in mid-November and I have to say, the last few months went by very fast. Overall, I am happy to report that we finished everything that had been planned for this collaboration. 
+
+Most of the work is targeted at improving the Vulkan backend and will benefit all platforms that use Vulkan. The list of improvements is too long to list here, but some highlights are:
+
+- Usage of Unified Memory Architecture (UMA) buffers when available
+- Add Android [Thermal API](https://developer.android.com/games/optimize/adpf/thermal) support
+- Replace large push constants with dynamic uniform buffers
+- Optimize descriptor sets and descriptor set batching
+- Optimize swapchain operations
+- Integrate [Swappy frame pacing](https://developer.android.com/games/sdk/frame-pacing) from the Google AGDK
+
+During the project, we tested two different 3D scenes using a Google Pixel 7 and a Samsung S23. Depending on the project and device, we see a consistent 10% - 20% reduction in GPU frame times. Since mobile devices are currently heavily GPU bottlenecked in 3D scenes on mobile devices this translates directly into a 10%-20% frame time improvement!
```

Comment:
```suggestion
During the project, we tested two different 3D scenes using a Google Pixel 7 and a Samsung S23. Depending on the project and device, we see a consistent 10% - 20% reduction in GPU frame times. Since mobile devices are currently heavily GPU bottlenecked in 3D scenes this translates directly into a 10%-20% frame time improvement!
```